.github/workflows/docker-compose-tests.yml

```diff
@@ -24,6 +24,13 @@ jobs:
       - name: Run tests against database container
         run: docker-compose -f docker-compose.yml -f tests/docker-compose-test.yml up --build --exit-code-from backend
 
+      - name: Setup upterm session
```

Comment:
Yes, it gives you ssh access to the VM where you can run the docker stuff in the action environment. If you connect, it keeps the VM up and running until there's activity. If nobody connects within 5 minutes it just destroys it. 
I was going to remove it at first, but I don't think it hurts us and it can be useful to have. 
What do you think?

Reply:
Sounds interesting. Yes, we can see how it goes having it